packages/code-du-travail-frontend/pages/kali.js

```diff
@@ -50,8 +49,8 @@ class Kali extends React.Component {
           title={title}
           emptyMessage="Cette convention collective n'a pas été trouvée"
           footer="Informations fournies par la DILA"
-          sourceType="Convention collective"
-          icon={ArticleIcon}
+          icon={null}
+          narrow={false}
```

Comment:
same here bool value in jsx (and html) 
no prop =>  false 
`narrow` => true

and in Answer component,  narrow="false" to be sure it default to no-value